org.eclipse.sprotty/src/main/java/org/eclipse/sprotty/Actions.xtend

```diff
@@ -502,6 +502,7 @@ class RequestExportSvgAction implements RequestAction<ExportSvgAction> {
 	public static val KIND = 'requestExportSvg'
 	String kind = KIND
 	
+	Map<String, String> options
```

Comment:
Thinking again, it might be actually better to model the options object explicitly, i.e. with its own Xtend class.

Reply:
I have done that; similar to Sprotty frontend